CMake/itkFFTImageFilterInitFactoryRegisterManager.h.in

```diff
@@ -49,7 +44,10 @@ class FFTImageFilterFactoryRegisterManager
 // application translation units. Note that this code will be expanded in the
 // ITK-based applications and not in ITK itself.
 //
-const FFTImageFilterFactoryRegisterManager FFTImageFilterFactoryRegisterManagerInstance;
+void (* const FFTImageFilterInitFactoryRegisterRegisterList[])(void) = {
```

Comment:
A question here -- if wrapping no longer uses static factory initialization, how can we verify that remaining factory static initialization works for C++ projects? Do we need to set up a separate project that can consume ITK for the purpose of this testing? I don't know that it is critical for getting this patch out, but we should at least consider test coverage implications.

Reply:
Yes, we have a simple "HelloWorld" project test that verifies that we can build against ITK. Similar should be added that exercise the static factory initialization.

https://github.com/InsightSoftwareConsortium/ITK/tree/master/Examples/Installation

Reply:
Is this built and run as part of CI? (not necessarily with every PR, nightly would be fine)

Reply:
If I recall correctly, it is enabled in CI where the Examples are enabled.

Reply:
I am not sure we have any nightly machines with examples enabled. CI doesn't do it either, right?

Reply:
It was at least tested on the nightlies in the past.